charts/grafana/templates/image-renderer-deployment.yaml

```
@@ -90,11 +90,10 @@ spec:
             - name: {{ $key | quote }}
               value: {{ $value | quote }}
           {{- end }}
+      {{- with .Values.imageRenderer.containerSecurityContext }}
           securityContext:
-            capabilities:
-              drop: ['all']
-            allowPrivilegeEscalation: false
-            readOnlyRootFilesystem: true
+{{ toYaml . | nindent 12 }}
+      {{- end }}
```

Minor indent change.
```suggestion
      {{- with .Values.imageRenderer.containerSecurityContext }}
          securityContext:
            {{- toYaml . | nindent 12 }}
      {{- end }}
```